config/app.yaml

```diff
@@ -0,0 +1,13 @@
+env :
+  production :
+  staging :
+  development :
+    db :
+#      net  : "tcp"        # tcp|sock
+#      host : "127.0.0.1"  # not required for net:sock
+#      port : 3306
+      name : "myBusiness"
+      user : "root"
+      pass : "rootpassword"
```

Comment:
we should force users to enter password via the command line when running the app, rather than store in plaintext. this can be left as an option here, but throw a warning/prompt that requires over-riding. w see also #4, which is intended to add a hidden password prompt when running the app